### PR TITLE
feat: add typography

### PR DESCRIPTION
**Pull request by afikurkhan-dsi**

### CODE REVIEW

**Comment by afikur** [comment]
can you check it again

File: style.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+h1 {
+    font-size: 2.5em;
+}
+
+h2 {
+    font-size: 2em;
+}
+
+h3 {
```
Comment (afikur) [inline style.css]: change font size to 2em

File: style.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    font-family: Arial, sans-serf;
+    font-size: 16px;
+}
+
+h1 {
+    font-size: 2.5em;
+}
+
+h2 {
+    font-size: 2em;
```
Comment (afikur) [inline style.css]: change this to 3em

Comment (afikurkhan-dsi) [reply style.css]: ok

**Pull request merged**